schema/locations/resolvers.ts

```diff
@@ -739,6 +739,92 @@ const resolvers: Resolvers = {
         monsters: [],
       };
     },
+    async moveAStar(parent, args, context: BaseContext): Promise<MoveResponse> {
+      // Ima just comment everything I did that is different from move
+
+      if (!context?.auth?.id) {
+        throw new ForbiddenError("Missing auth");
+      }
+
+      const hero = await context.db.hero.get(context.auth.id);
+      const account = await context.db.account.get(context.auth.id);
+
+      if (hero.combat.health <= 0) {
+        throw new UserInputError("You cannot move while dead!");
+      }
+
+      const heroLocation = { ...hero.location };
```

Comment:
"startLocation" might be a better name here

Reply:
Changed